jlm/hls/Makefile.sub

```diff
@@ -47,15 +49,18 @@ libhls_HEADERS = \
 	jlm/hls/backend/rvsdg2rhls/add-triggers.hpp \
 	jlm/hls/backend/rvsdg2rhls/alloca-conv.hpp \
 	jlm/hls/backend/rvsdg2rhls/check-rhls.hpp \
+	jlm/hls/backend/rvsdg2rhls/dae-conv.hpp \
 	jlm/hls/backend/rvsdg2rhls/DeadNodeElimination.hpp \
 	jlm/hls/backend/rvsdg2rhls/GammaConversion.hpp \
 	jlm/hls/backend/rvsdg2rhls/mem-queue.hpp \
 	jlm/hls/backend/rvsdg2rhls/mem-sep.hpp \
 	jlm/hls/backend/rvsdg2rhls/memstate-conv.hpp \
 	jlm/hls/backend/rvsdg2rhls/remove-redundant-buf.hpp \
 	jlm/hls/backend/rvsdg2rhls/remove-unused-state.hpp \
+	jlm/hls/backend/rvsdg2rhls/rhls-dne.hpp \
 	jlm/hls/backend/rvsdg2rhls/rvsdg2rhls.hpp \
 	jlm/hls/backend/rvsdg2rhls/ThetaConversion.hpp \
+	jlm/hls/backend/rvsdg2rhls/UnusedStateRemoval.hpp \
```

Comment:
This file does not exist in the PR.

Reply:
Okay, this file already exists in the repository, but is just not mentioned here.

Reply:
Ignore the above comments.